WebAPI/WebAPI/Controllers/UsersController.cs

```diff
@@ -21,6 +22,7 @@ public UsersController(IUserRepository repository)
             userRepository = repository;
         }
 
+        [Authorize]
```

Comment:
better to use a global authorization filter(then all your routes will require authorization by default) and just make login action allows for anonyms